action.yaml

```
@@ -9,19 +9,37 @@ inputs:
 runs:
   using: "composite"
   steps:
-  - name: Determine Ubuntu Version
+  - name: Determine Cache Key
```

```suggestion
  - name: Determine cache options
```

Nitpick, but maybe we also want to indicate here that it's not only about the cache key but also the path.

---

```suggestion
  - name: Determine Cache Options
```